hibernate-core/src/main/java/org/hibernate/dialect/DerbyDialect.java

```diff
@@ -270,39 +288,35 @@ public String translateExtractField(TemporalUnit unit) {
 	 * type, but it doesn't know how to cast to it. Worse,
 	 * Derby makes us use the {@code double()} function to
 	 * cast things to its floating point types.
+	 * @param from
+	 * @param to
```

Comment:
if you're going to add these, you need to add descriptive text.

Reply:
Thanks, I'll remove that.